README.md

```diff
@@ -4,6 +4,11 @@ API for serializing and de-serializing JavaScript objects using the
 addon compared to the native `JSON` object isn't too bad, and the space
 required for serialized data is far less than JSON.
 
+### Extension: support for undefined
+The standard msgpack protocol turns `undefined` into `null`.
+This branch adopts (creationix's extension)[https://github.com/creationix/msgpack-js-browser#extension]
+for `undefined` and uses for it the reserved value '0xc4'
```

Comment:
Also, I'm almost certain it's a bad idea to use this.  The spec is here: https://github.com/msgpack/msgpack/blob/master/spec.md
`0xd4` (and `0xc4`) are clearly in use for other things.  This could break packing to another library, or unpacking from another library.  Unless there is a region of extension types for user defined types, or a real undef type, I think this is very dangerous to add.  Obviously, msgpack has chosen to treat undefined as nil.  Let me know if I'm wrong, I just did a few minutes of research.